bin/generate_participant.py

```diff
@@ -1 +1,29 @@
-1
+from pathlib import Path
+Path.cwd()
+data = Path.cwd()
+data.absolute() # Prints absolute path
+data.glob("Sub_*") # Selects all directories named Sub_* In Python, the glob module is used to retrieve files/pathnames matching a specified pattern.
+list(data.glob("Sub_*")) # Lists directories
+subj = list(data.glob("Sub_*")) # Adds variable for list   
+    
+
+sub_str = [ str(e) for e in subj] # subjects as string, converts elements from windows path to string retaining list format
+
+    
+len(sub_str) # Equals amount of elements in substr list
+total_elements = len(sub_str) # Variable for while loop
+element = 0 # List elements start at 0
+
+while element <= total_elements-1: # total_elements = 4 in this instance
+
+    sub_str[element] = sub_str[element].replace("\\","").replace("'","").replace(":","") # Removes unwanted symbols
+    sub_str[element] = sub_str[element].replace("CUsersjoelpDesktopData","") # Removes unwanted text
```

Comment:
Formatting: space after `,`
This is a good solution for your system, but the downside is only Windows uses double backslashes as path separators, and not all users are named Joel :laughing: . You will eventually learn what is specific to Windows and start to write more generalisable code. 
Since `sub_str[element]` is a string, there are lots of default python string related functions for use.
I would use the split function, so it became something like this
```python
import os  # add this to the top of the file not in the loop
# split string by os specific separator, return a list of strings
cur_sub = sub_str[element].split(os.sep) 
# save the last item (Sub_*). We know the last item is the subject ID because line 7
sub_str[element] = cur_sub[-1] 
```